friends/src/actions/index.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+export const GET_FRIENDS = 'GET_FRIENDS';
+export const FETCHING_FRIENDS = 'FETCHING_FRIENDS';
+export const ERROR = 'ERROR';
+export const SAVING_FRIENDS = 'SAVING_FRIENDS';
+export const FRIENDS_SAVED = 'FRIENDS_SAVED';
+
+export const getFriends = () => {
+  const promise = axios.get('http://localhost:5000/api/friends');
```

Comment:
One thing I'd recommend is creating a variable for the API URL since it's something you will be using in multiple places often. So instead of `axios.get('http://localhost:5000/api/friends');` you could use something like `axios.get(`${URL}/get`);` which you can then also use for your `axios.post` in your `addFriend` function.